_posts/0000-01-02-liammccon.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
To add is to take away from what is not added 
_To italicise is to take away from what has not been italicised?_
**to bold?**